ios/Classes/AppodealBannerFactory.swift

```diff
@@ -10,14 +10,40 @@ import Flutter
 import Foundation
 
 class AppodealBannerFactory: NSObject, FlutterPlatformViewFactory {
-    func create(withFrame _: CGRect, viewIdentifier _: Int64, arguments _: Any?) -> FlutterPlatformView {
-        AppodealBannerView()
+    var instance: SwiftAppodealFlutterPlugin
+
+    init(instance: SwiftAppodealFlutterPlugin) {
+        self.instance = instance
```

Comment:
I don't have much to say here, I just wanted to understand the motivation behind passing an instance of `SwiftAppodealFlutterPlugin` when we create a banner 🙂

Reply:
It's used a bit further down in this file when setting the `banner.delegate`, since `SwiftAppodealFlutterPlugin` has the `AppodealBannerViewDelegate` extension. As @nixxxon said we're a bit new to Swift so not sure if there is a better way of doing it 🙂